pkg/kubelet/container_manager_linux.go

```diff
@@ -142,26 +142,48 @@ func createManager(containerName string) *fs.Manager {
 	}
 }
 
-const sysctlVmOvercommitMemory = "vm/overcommit_memory"
-
-// disableKernelMemoryOvercommitHandling tells the kernel to perform no memory over-commit handling.
-// Under this setting, the potential for memory overload is increased, but so is performance for
-// memory-intensive tasks
-// sets /proc/sys/vm/overcommit_memory to 1
-func disableKernelMemoryOvercommitHandling() error {
-	val, err := utilsysctl.GetSysctl(sysctlVmOvercommitMemory)
-	if err != nil {
-		return err
-	}
-	if val == 1 {
-		return nil
+// TODO: plumb this up as a flag to Kubelet in a future PR
+type KernelTunableBehavior string
```

Comment:
I am not sure how complicated I want to make it in the future.  

Our ops guys just said "I don't like processes that set kernel flags without telling me".  

I think Warn is useful for people that are running hack/local-up-cluster.sh where they don't want to change the kernel flags on their host dev machine so I don't think its worth getting too fancy.